src/environment/stella_environment.cpp

```diff
@@ -195,6 +203,26 @@ bool StellaEnvironment::isTerminal() const {
      m_state.getEpisodeFrameNumber() >= m_max_num_frames_per_episode));
 }
 
+void StellaEnvironment::pressSelect(size_t num_steps) {
+  Event* event = m_osystem->event();
+  m_state.pressSelect(event);
+  for (size_t t = 0; t < num_steps; t++) {
+    m_osystem->console().mediaSource().update();
+  }
+  processScreen();
+  processRAM();
+  emulate(PLAYER_A_NOOP, PLAYER_B_NOOP);
+}
+
+void StellaEnvironment::setDifficulty(difficulty_t mask) {
```

Comment:
This 'mask' variable name doesn't do it for me. Can we call it 'difficulty' or 'value'? 'Mask' suggests that we need to know the internal details of the integer, in which case I would split it into two arguments (left_difficulty and right_difficulty).

Reply:
I changed `mask` to `value`.